sources/zsh/history

```diff
@@ -314,8 +314,11 @@ certipy 'domain.local'/'user':'password'@'domaincontroller' -dc-ip 'domaincontro
 echo '8.8.8.8' | hakrevdns
 prips 192.168.0.0 192.168.0.255
 prips -i4 192.168.0.10 192.168.0.250
-prips 173.0.84.0/24 | hakrevdns
-nuclei -u https://example.com
+prips 173.0.84.0/24 | hakrevdns -d
+cat domain.txt |naabu -pf ports -c 10 -silent |anew open_ports.txt
+cat open_ports.txt |httpx -t 10 -silent|anew webservers.txt
+cat webservers.txt|nuclei -silent -max-host-error 10 -t /root/nuclei-templates/exposed-panels -rl 15 -stats -o panels.txt
+cat webservers.txt|nuclei -silent -max-host-error 10 -tags tomcat,jboss,ibm -rl 15 -stats -o app.txt
```

Comment:
For homogeneity sake, can you make sure there is a space before and after each pipe, and also opt for long args whenever possible and necessary (i.e. not needed for short args that are really common like `-t`, `-o`)?